hail/python/hail/vds/impex.py

```diff
@@ -0,0 +1,335 @@
+
+    extra_header = (
+        '##SVCR="This is an VCF that implements hail\'s Scalable Variant Call '
+        'Representation. See https://doi.org/10.1101/2024.01.09.574205 for more '
+        'information."\n'
+    )
+    if VariantDataset.ref_block_max_length_field in ref.globals:
+        rbml = hl.eval(ref[VariantDataset.ref_block_max_length_field])
+        extra_header += f'##ref_block_max_length={rbml}\n'
+
+    fs = hl.current_backend().fs
+    if append_to_header:
+        with fs.open(append_to_header, 'r') as header_file:
+            extra_header += header_file.read()
+
+    new_header_file_path = hl.utils.new_temp_file('vds-append-to-header', 'txt')
+    with fs.open(new_header_file_path, 'w') as new_header_file:
+        new_header_file.write(extra_header)
+
+    vcf = to_merged_sparse_mt(VariantDataset(reference_data=ref, variant_data=var))
+    hl.export_vcf(vcf, output, append_to_header=new_header_file_path, parallel=parallel, metadata=metadata, tabix=tabix)
+
+
+@typecheck(
+    path=oneof(str, sequenceof(str)),
+    is_split=bool,
+    ref_block_fields=sequenceof(str),
+    infer_ref_block_fields=bool,
+    force=bool,
+    force_bgz=bool,
+    header_file=nullable(str),
+    min_partitions=nullable(int),
+    call_fields=oneof(str, sequenceof(str)),
+    reference_genome=nullable(reference_genome_type),
+    contig_recoding=nullable(dictof(str, str)),
+    array_elements_required=bool,
+    skip_invalid_loci=bool,
+    entry_float_type=enumeration(tfloat32, tfloat64),
+    filter=nullable(str),
+    find_replace=nullable(sized_tupleof(str, str)),
+    n_partitions=nullable(int),
+    block_size=nullable(int),
+    _create_row_uids=bool,
+    _create_col_uids=bool,
+)
+def import_vcf(
+    path,
+    *,
+    is_split=False,
+    ref_block_fields=(),
+    infer_ref_block_fields=True,
+    force=False,
+    force_bgz=False,
+    header_file=None,
+    min_partitions=None,
+    call_fields=('LPGT', 'PGT'),
+    reference_genome='default',
+    contig_recoding=None,
+    array_elements_required=False,
+    skip_invalid_loci=False,
+    entry_float_type=tfloat64,
+    filter=None,
+    find_replace=None,
+    n_partitions=None,
+    block_size=None,
+    _create_row_uids=False,
+    _create_col_uids=False,
+) -> VariantDataset:
+    """Import SVCR-VCF file(s) as a :class:`.VariantDataset`.
+
+    .. include:: _templates/experimental.rst
+
+    Notes
+    -----
+    The internal implementation of this method is a combination of :func:`hail.import_vcf` and
+    :func:`VariantDataset.from_merged_representation`. Refer to the documentation of those methods
+    for detailed usage of the parameters of both. The ``drop_samples`` parameter from
+    :func:`hail.import_vcf` is not present as a 'sites only' VDS is nonsense.
+
+    Note
+    ----
+    The following validations and transformations take place:
+
+    #. The ``LEN`` FORMAT field must exist and be of type :py:data:`.tint32`
```

Comment:
Should the types listed in `FORMAT` and `INFO` sections be vcf types rather than hail types?